MapleServer2/PacketHandlers/Game/UserChatHandler.cs

```diff
@@ -33,34 +35,62 @@ public override void Handle(GameSession session, PacketReader packet)
                 return;
             }
 
+            Packet itemLinkPacket = null;
```

Comment:
Can all of this be put in a separate method that takes in a message and returns `itemLinkPacket` or null?

Reply:
if you like that better? sure

Reply:
oh wait you mean the whole item linking part in a function?
I don't see the benefit of it, since it is only used once

Reply:
Yeah it's only being used once but it's an isolated functionality, getting the item link packet, it would read better in a separate function even if that function will only be called once

Reply:
yea reading is for sure better, i'm always just concerned about the overhead of the functioon call.
But if you think it's worth it, i'll certainly do it